tests/runtime/TestStlFunctions.hip

```diff
@@ -51,7 +51,9 @@ template <typename T, typename FnT> void launchBinaryFn(FnT Fn) {
 
 int main() {
 
+#if 0
   launchUnaryFn<float>([] __device__(auto x) { return std::abs(x); });
+#endif
```

Comment:
I'm getting warnings about this being ambiguous, I can add a log with these errors later.
Why isn't this an issue for other platforms?

Reply:
> I can add a log with these errors later.

Please, send us the log.